### DISCUSSION TITLE
Running Audio.sh gets stuck at "Download Attempt number 1"

### DISCUSSION THREAD

**Discussion by olimeca** [RandomNinjaAtk/docker-lidarr-extended#203] [General]
Using latest image and running Audio.sh from within the container it gets to the point where it tries to download and just stays there.

**Comment by RandomNinjaAtk** [comment]
You should check the running processes to check if the download client is running.... If it is, then everything is working as expected...

The issue then might be related to your specific configuration or environment. I do not provide support for the clients... so you may need to do some manual testing  and possibly report the issue upstream...

Edit: If it also only haults for one particular album, you could always unmonitor it to effectively skip it...

**Reply by olimeca** [reply]
I tried blowing my entire config folder out and recreating the container from scratch, still the same issue. I also tried different albums/singles and they all get to the "Download Attempt number 1" point and it just stays there. Checking htop I do see /usr/bin/deemix...etc running but nothing ever actually downloads.

Kind of stumped, been using this for a long time and it just started happening. I will try to mess around with it more and see if I can make any progress.

**Reply by RandomNinjaAtk** [reply]
Maybe its just something wrong with that particular album. Some things you should try, unmonitor that album so it is skipped, update your ARL token...

**Reply by olimeca** [reply]
Tried multiple albums and they all do the same thing. Tested the arl with deemix by itself and was able to download without issue. I am using the compose from the dockerhub minus a few settings that didn't pertain to me. I don't know what else in my setup that would possibly cause this to stop working. Very strange.

**Reply by RandomNinjaAtk** [reply]
Maybe its a connection issue, if your using a VPN or something else... Honestly have no idea, and if it was a bigger problem, there would be a lot more reports. So it must be something unique to your particular configuration/setup.... If the dl client is running, then the script has done its job and it's an issue with the client at that point...

**Reply by olimeca** [reply]
FWIW I checked the Lidarr discord even though it is not the place for Lidarr-extended support and someone posted they are having the same problem as me this week using Lidarr-extended.

"The deezer / deemix download script says this : 2023-02-10 19:27:57 :: Audio :: 1.0.309 :: 1 :: missing :: 1 of 1 :: Nicholas Hooper :: Harry Potter and the Order of the Phoenix (Original Motion Picture Soundtrack) :: Album :: Download Attempt number 1 

but doesn't do anything else after that. Nothing is downloading."

Anyways thank you for the help. I will wait and see if more users end up running into the same issue.

**Reply by RandomNinjaAtk** [reply]
What is the musicbrainz release group id? I could at least see if I could replicate... Everything appears to be working in my setup, so without being able to replicate, can't begin to troubleshoot further...

**Reply by olimeca** [reply]
Definitely hard to troubleshoot something when it can't be replicated, totally get it.

The MBID for the track I am having issues with is 95ff5306-2b74-46c4-9f7a-a9d5c439efa6

However it is happening for all albums/tracks so I am not sure how much help that will be.

**Reply by RandomNinjaAtk** [reply]
Hmm, turns out I already have that release... I'll try the Harry Potter one from your Logs...

**Reply by RandomNinjaAtk** [reply]
I had no issues with both albums from that artist (musicbrainz artist id: b1ea49d6-e25b-4675-9104-8329442c3e9e)... So it must be unique to your setup.

**Reply by olimeca** [reply]
I’ll keep poking at it and see if I can figure it out. Thanks for trying.

**Reply by RandomNinjaAtk** [reply]
Maybe try changing your download quality settings. If you were set for lossless, maybe go for the lossy format.
